README.md

```diff
@@ -25,3 +25,41 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+-----------------------------
+
+This can be split up into multiple parts using T(n) where n is the size of the array, when n is less than or equal to 2 and when it's greater than 2.
+When it is less than or equal to 2 it is constant since it will return 0, the first element, or the just the first and second added together.
+However when n is greater than 2 it needs to split into 3 parts which is done at the same time everytime plus the recursive calls, since it is 3 recursive calls in 3 parts it can be represented with 3T(n/3) where it's 3T(n) for the calls, and (n/3) for the 3 parts.
+
+This gives us an equation of T(n) = 3T(n/3) + n
```

Comment:
In your code it looks like you're only summing up the partial results after the recursive calls, i.e. something that's constant time.